### ISSUE TITLE
GUI not opening on Linux after upgrading from 9.x to 10.17

### ISSUE THREAD

**Issue by Mik-S-UMS** [UniversalMediaServer/UniversalMediaServer#2871]
There is a user on the forum that can no longer open the GUI after upgrading. 
https://www.universalmediaserver.com/forum/viewtopic.php?f=9&t=15027

I can't see any reason in the logs but there are several errors that will need looking at. I don't see how they can effect the GUI though but may be relevant to recent refactoring.

**Comment by SubJunk** [comment]
I think I see the problem, I'll post a build for that user. Thanks for posting this here and for your work to get those logs!

**Comment by dln949** [comment]
I'm the person who originally asked about the problem on the UMS forum.  

Thanks for doing this.  

I am sorry, but I am a novice, I must ask these two questions:  

1)  Briefly, in layman's terms, what was the problem?  

2)  Exactly what am I supposed to do now so I don't have this problem?  Is there now a new version of UMS at the UMS download web page that I need to download????

**Comment by Mik-S-UMS** [comment]
He has reported that the GUI is still not opening (but the errors in logs are gone)
I think he might need a newer version of openjdk but I'm just guessing. Any Ideas?

**Comment by Mik-S-UMS** [comment]
Seems like a problem in 10.5.0, 10.4.1 is working for him.

**Comment by SubJunk** [comment]
@Mik-S-UMS I don't think the java version will change anything since we bundle java for his operating system. I'll look at the code changes in 10.5.0. Thanks for communicating that in here, it's very helpful
